js/wp-seo-replacevar-plugin-302.js

```diff
@@ -42,11 +53,68 @@
 			data = this.parentReplace( data );
 			data = this.doubleSepReplace( data );
 			data = this.excerptReplace( data );
+			data = this.categoryReplace( data );
+			data = this.customTaxonomyReplace( data );
+			data = this.tagReplace( data );
+
+			// Taxonomy replacements
+			if ( jQuery( 'input[name=tag_ID]').val() !== undefined ) {
```

Comment:
Please move this to a new function or use a property passed by the PHP to indicate on which page we are.